app/instance-initializers/configure-trackjs.js

```diff
@@ -1,34 +1,52 @@
 import Ember from 'ember';
 
 export function initialize(application) {
-  let trackJs = application.container.lookup('service:trackjs');
-  let appVersion = application.container.lookup('application:main').get('version');
+  const trackJs = application.container.lookup('service:trackjs');
 
   trackJs.configure({
-    version: appVersion
+    version: application.container.lookup('application:main').get('version')
   });
 
-  // http://docs.trackjs.com/Examples/Integrating_with_Ember
-  Ember.onerror = function (err) {
-    if (err.name === 'TransitionError') {
-      Ember.debug('Ignoring TransitionError:', err);
-    } else {
-      trackJs.track(err);
-      Ember.Logger.assert(false, err);
+  Ember.onerror = error => {
+    // If you pass a POJO to TrackJS, you'll see "[object Object]"
+    // in reports instead of object content. :(
+
+    // If the error is an Error object, we pass it directly.
+    if (error instanceof Error) {
+      trackJs.track(error);
+      Ember.Logger.error(error.stack);
+      return;
     }
-  };
 
-  Ember.RSVP.on('error', function (err) {
-    if (err.name === 'TransitionError') {
-      Ember.debug('Ignoring TransitionError:', err);
-    } else {
-      trackJs.track(err);
-      Ember.Logger.assert(false, err);
+    // Otherwise, we have to serialize it manually in order
+    // to see something better than "[object Object]".
+    let serializedError;
+    try {
+      // This will work for both primitives and POJOs:
+      serializedError = JSON.stringify(error, null, 2);
```

Comment:
They tell it to produce a multiline JSON with two-space indentation, rather than cluttered into a one-liner. I've tried it and TrackJS UI deals nicely with mutliline messages, removing linebreaks when it shows them as one-liners.